src/mistral_common/tokens/tokenizers/tekken.py

```diff
@@ -203,6 +204,7 @@ def __init__(
         self._audio_config = audio_config
 
         self._all_special_tokens = special_tokens
+        self._special_token_ids = {t["rank"] for t in special_tokens}
         self._special_tokens_reverse_vocab = {t["token_str"]: t["rank"] for t in special_tokens}
         self._vocab = [self.id_to_piece(i) for i in range(vocab_size)]
```

Comment:
These are generally accessed in certains ways by vLLM or Transformers integrations.

Can be done later but we should add getters.

Reply:
Would like to try to not have to access them actually, but ok let's see